src/Sylius/Bundle/UserBundle/Form/UserVerificationTransformer.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\UserBundle\Form;
+
+use Sylius\Bundle\UserBundle\Form\Type\UserType;
+use Sylius\Component\User\Model\User;
+use Symfony\Component\Form\DataTransformerInterface;
+
+class UserVerificationTransformer implements DataTransformerInterface
+{
+    public function transform($data)
+    {
+        return (bool) $data;
```

Comment:
```suggestion
        return $data instanceof \DateTimeInterface;
```

Reply:
Transform function is changing the modelData to viewData, as I am using on view Checkbox ( or toogle to be precise) I need to have true or false viewData.

Reply:
What I wanted to achieve is to be more explicit about what is happening here. `$data instanceof \DateTimeInterface` would return true if there is a DateTime object, false otherwise. Maybe comparision to null would be a better choice. I'm just struggling with the direct casting of DateTime object to (bool)